RELEASE.md

```diff
@@ -47,6 +47,12 @@
 * **Follow all of the steps outlined externally in `DOCUMENTATION.md`**,
   notably so that the correct Sphinx-related environment is prepared for
   documentation building.
+  
+* Ensure that the [PDF for Cheat Sheet](docs/_downloads/cheatsheet.pdf) 
+  is updated to include any API changes. The PDF is created using Canva 
+  keeping in mind the colours and fonts of the website. The same could 
+  be edited using this 
+  [link](https://www.canva.com/design/DAFk9_BVfNY/gmQHycBiV_YbTIWMqYxK1g/edit).
```

Comment:
Nicely clarified here.

Though I am wondering, how would we access the current version of the A4/printable cheat sheet before it is exported as PDF? Is there an option on Canva to save the cheat sheet in editable/build form so that it can be edited and re-exported and saved as PDF when we want to do that, and if so please can we save that also to the repo? Better to have it stored on GitHub than having to possibly pester you for such a resource anytime it needed an update, or re-write the cheat sheet from scratch each time for lack of one, if that makes sense!

Reply:
Hi Sadie, Sorry, I didn't get what you are trying to explain! The given link itself contains the Cheat Sheet in an editable form which can be exported as a PDF, PNG or any format of choice.